diff --git a/dist/generate-html.js b/dist/generate-html.js
index d98bd1784c8433525bbf0d0888c3eb95fe1cc620..fe0e0afe0eae60f29ce31cd2b7796e7ac7a8eaf0 100644
--- a/dist/generate-html.js
+++ b/dist/generate-html.js
@@ -1,4 +1,4 @@
-import { getStyles, getScripts, getLogo } from './assets-inline';
+import { getStyles, getScripts, getLogo } from './assets-inline.js';
 /**
  * Generate beautiful HTML documentation from route information
  * @param routes - Array of route information
diff --git a/dist/index.js b/dist/index.js
index 6eedbc93d82addedca6fd76cad31811f644dc9f5..c581a5d8f598256950c25befdce8f75f542764cf 100644
--- a/dist/index.js
+++ b/dist/index.js
@@ -6,5 +6,5 @@
  *
  * @packageDocumentation
  */
-export { collectRoutes } from './collect-routes';
-export { generateDocsHtml } from './generate-html';
+export { collectRoutes } from './collect-routes.js';
+export { generateDocsHtml } from './generate-html.js';
diff --git a/package.json b/package.json
index c60968cf99530809db5d83f70f048374c77f4421..2845d13709880d6905939505f11983f612013a5c 100644
--- a/package.json
+++ b/package.json
@@ -22,6 +22,7 @@
   },
   "exports": {
     ".": {
+      "require": "./dist/index.js",
       "import": "./dist/index.js",
       "types": "./dist/index.d.ts"
     }
